spark3-extensions/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/CreateTableEncKey.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.sql.catalyst.plans.logical
+
+import org.apache.spark.sql.catalyst.expressions.Attribute
+
+case class CreateTableEncKey(
+    table: Seq[String],
```

Comment:
Is there a reason to prefer `Seq[String]` here? Would it possibly make more sense to use an identifier type here, such as `org.apache.spark.sql.connector.catalog.Identifier`?

If other ones use `Seq[String]` or if there's a reason that we can't use a more definitive type, feel free to close this comment =)

Reply:
EDIT: I took a look, and it seems that `Seq[String]` is the standard way of doing it in some of the others as well (plus you use spark catalog `Identifier` class in the implementation.

Please feel free to resolve these comments =)

Reply:
I will leave this as is :)